ivy_tests/test_ivy/test_functional/test_core/test_linalg.py

```diff
@@ -1257,6 +1321,60 @@ def test_cholesky(
     )
 
 
+# cov
+@handle_test(
+    fn_tree="functional.ivy.cov",
+    dtype_x1_x2=_get_dtype_value1_value2_for_cov(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        max_num_dims=2,
+        min_dim_size=2,
+        max_dim_size=5,
+        min_value=-1e10,
+        max_value=1e10,
+        abs_smallest_val=0.01,
+        large_abs_safety_factor=2,
+    ),
+    num_positional_args=helpers.num_positional_args(fn_name="cov"),
+)
+def test_cov(
+    *,
+    dtype_x1_x2,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    fn_name,
+    container_flags,
+    instance_method,
+    backend_fw,
+    on_device,
+    ground_truth_backend,
+):
+    dtype, x1, x2, rowVar, bias, ddof, fweights, aweights = dtype_x1_x2
+    helpers.test_function(
+        ground_truth_backend=ground_truth_backend,
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        container_flags=container_flags,
+        instance_method=instance_method,
+        fw=backend_fw,
+        fn_name=fn_name,
+        on_device=on_device,
```

Comment:
You may configure the tolerance by adding, eg, 

```
rtol=1e-01,
atol=1e-01,
```

as arguments in the `test_function`